src/sanitizer.js

```diff
@@ -308,7 +326,17 @@ export function sanitizeHtml(html) {
  */
 export function sanitizeFormattingHtml(html) {
   return htmlSanitizer.sanitizeWithPolicy(html,
-      function(tagName, unusedAttrs) {
+      function(tagName, attribs) {
```

Comment:
nit: Not sure the name `attribs` conforms with the style guide [go/es6-style#naming-rules-common-to-all-identifiers](https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md#naming-rules-common-to-all-identifiers)

Reply:
These are the argument names in the 3rd party library that uses this function as a callback, so I figured they should be the same. I can change them if you still think it prudent

Reply:
👍 nope it makes sense to follow the library